docs/embedding.md

```diff
@@ -0,0 +1,72 @@
+
+See it in action:  
+[![Edit Custom Webchat Options](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/embedding-the-cognigy-webchat-4xkv8?fontsize=14&hidenavigation=1&theme=dark)
+
+### Endpoint Settings
+| Name | Type | Default | Description |
+| - | - | - | - |
+| startBehavior | 'none' , 'button', 'injection' | 'none' | If 'none', will start the webchat with a text input, 'button' will display a get started button with a preconfigured message, 'injection' will automatically send a message to the bot. 
+| getStartedText | string | "Get Started" | The text to display in the Webchat when clicking the Get Started Button / when sending the auto message. | getStartedPayload | string | "GET_STARTED" | The payload to send to your Flow when clicking the Get Started Button / when sending the auto message. 
+| getStartedButtonText | string | "GET_STARTED" | The text to display on the Get Started Button / when sending the auto message. 
+| inputPlaceholder | string | "Write a reply" | The placeholder text to display in the input field. 
+| enableSTT | boolean | false | Whether to enable speech input that lets the user speak to the Webchat instead of only typing. 
+| enableTTS | boolean | false | Whether to enable the browser to read the bot messages aloud. 
+| colorScheme | string | The background color of the header and bot messages in the Webchat. | designTemplate | 1 or 2 | 1 | The Webchat design template to use. We default to design template 1 (bottom right with a button), you can switch to template 2, which is the centered webchat. 
+| messageLogoUrl | string | COGNIGY.AI Logo | The logo to display next to the messages from your bot in the Webchat. Defaults to a COGNIGY.AI logo. 
+| headerLogoUrl | string | COGNIGY.AI Logo | The logo to display in the header of the Webchat. Defaults to a COGNIGY.AI logo. 
+| enableConnectionStatusIndicator | boolean | false | Whether to show a warning if the connection is lost during a conversation. The warning will disappear when the connection is re-established. | enableTypingIndicator | boolean | true | Whether to enable typing indicators in the Webchat when the Conversational AI is replying. Requires a messageDelay to be set. 
+| enablePersistentMenu | boolean | false | Whether to enable the Persistent Menu 
+| persistentMenu | [Persitent Menu](#persistent-menu) | - | The Persistent Menu to render in the Webchat.
+
+See it in action:  
+[![Edit Override Endpoint Settings](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/embedding-the-cognigy-webchat-bpz1r?fontsize=14&hidenavigation=1&theme=dark)
+
+#### Persistent Menu
+| Name | Type | Default | Description |
+| - | - | - | - |
+| title | string | `""` | The title of the Persistent Menu |
+| menuItems | List of [Persitent Menu Item](#persistent-menu-item)s | `[]` | A List of Items that should appear in the Persistent Menu
```

Comment:
Typo "Persitent". And better move the s into the brackets, else it can look a bit silly

Reply:
fixed, and fixed.